main.go

```diff
@@ -40,6 +40,9 @@ const (
 // is set through linker options.
 var gitCommit string
 
+// durationBuckets is the buckets used for Prometheus histograms in seconds.
+var durationBuckets = []float64{.5, 1, 2.5, 5, 10, 20, 40, 80, 160, 320}
```

Comment:
Is this something worth mentioning in the README.md, or is it something obvious from the server side?

Reply:
I believe the default values 0-10secs is not very useful. Blobs are usually big and latency can easily go past 10 secs for big blobs >500MB, we have seen this in our dash on peak hours. This new default values is more realistic.

Reply:
I can of course mention on the README too.

Reply:
I don't have much experience with metrics dashboards- if it's something that would be obvious to people using these systems, then I don't think we need to document it.